definitions/private-law/json/CaseField.json

```diff
@@ -1278,5 +1174,69 @@
     "HintText": "Upload a scanned PDF or DOCX of the draft consent order, signed by both parties. Wherever possible, documents should be scanned in greyscale.",
     "RegularExpression": ".pdf,.docx",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "04/11/2021",
+    "CaseTypeID": "C100",
+    "ID": "IsCaseUrgent",
+    "Label": "Is this case urgent?",
+    "FieldType": "YesOrNo",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "04/11/2021",
+    "CaseTypeID": "C100",
+    "ID": "CaseUrgencyTimeAndReason",
+    "Label": "Set out how soon the case needs to be heard (days/hours), and the reason for urgency.",
```

Comment:
This doesn't match the copy on the figma screens - https://www.figma.com/file/M2WoOSgu8sRr0emyDWzf7Z/C100-Solicitor-prototype?node-id=1429%3A73092

Reply:
updated the copy for label